lib/win32/file.rb

```diff
@@ -60,7 +60,7 @@ def self.basename(file, suffix = nil)
 
     # Return a root path as-is.
     if PathIsRootW(wfile)
-      return file.tr(File::SEPARATOR, File::ALT_SEPARATOR)
+      return file.tr(File::SEPARATOR, File::ALT_SEPARATOR).gsub(/[\\]/,"/")
```

Comment:
You're changing forward slashes to back slashes, then back to forward slashes again? How about just:

`file.tr(File::ALT_SEPARATOR, File::SEPARATOR)`

May as well just convert all backslashes to forward slashes everywhere really.

Reply:
Thanks @djberg96, I have update the code also verified all working fine. Please review PR.